ext/drreg/drreg.h

```diff
@@ -345,6 +357,18 @@ drreg_status_t
 drreg_reserve_register(void *drcontext, instrlist_t *ilist, instr_t *where,
                        drvector_t *reg_allowed, OUT reg_id_t *reg);
 
+DR_EXPORT
```

Comment:
This comment is for line 349 which says the vector is for START_GPR..STOP_GPR.  That needs to be updated.

Reply:
I've added and amended the doxygen comments.